lib/ruby_lsp/handler.rb

```diff
@@ -162,29 +158,36 @@ def respond_with_document_highlight(uri, position)
       Requests::DocumentHighlight.run(store.get(uri), position)
     end
 
-    def configure_options(initialization_options)
-      @telemetry_enabled = initialization_options.fetch(:telemetryEnabled, false)
-    end
-
     def with_telemetry(request)
-      params = { request: request[:method], lspVersion: RubyLsp::VERSION }
       result = nil
+      error = nil
 
       request_time = Benchmark.realtime do
         result = yield
       rescue StandardError => e
```

Comment:
Question: Should we catch all kinds of errors here? For example, what if we get a `LoadError` if we've messed up an autoload path or something? Maybe we should rescue `Exception` instead?

Reply:
So, I tested and it looks like `Interrupt` is an `Exception`. I'm afraid swallowing it might make VS Code unable to stop the server if necessary. What do you think about adding `LoadError` to the rescued errors instead?

Reply:
Right, that's a good argument against handling `Exception`. Ok, let's leave it as is in that case, we should be able to catch `LoadError`s and others during test, hopefully.